mon-pix/app/components/account-recovery/confirmation-step.js

```diff
@@ -1,9 +1,15 @@
 import Component from '@glimmer/component';
 import { tracked } from '@glimmer/tracking';
+import { action } from '@ember/object';
 
 export default class ConfirmationStepComponent extends Component {
   @tracked checked = false;
 
+  @action
+  onChange(event) {
+    this.checked = !!event.target.checked;
```

Comment:
pourquoi faire le `!!` puisque checked est deja une boolean ?

Reply:
tu parles du this.checked ?

Reply:
je parle de `event.target.checked`, a mes souvenirs c'est deja reconnu comme une boolean alors le `!!` est redondant